docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -44,16 +44,19 @@ A conformant client library MUST make a GET request with the headers:
     * iOS MUST be `expo-platform: ios`.
     * Android MUST be `expo-platform: android`.
     * If it is not one of these platforms, the server SHOULD return a 400 or a 404
-2. `expo-runtime-version` MUST be a runtime version compatible with the client. A runtime version stipulates the native code setup a client is running. It should be set when the client is built. For example, in an iOS client, the value may be set in a plist file. 
+2. `expo-runtime-version` MUST be a runtime version compatible with the client. A runtime version stipulates the native code setup a client is running. It should be set when the client is built. For example, in an iOS client, the value may be set in a plist file.
 3. Any headers stipulated by a previous responses' [server defined headers](#manifest-response-headers).
 
 A conformant client library MUST also send at least one of `accept: application/expo+json` or `accept: application/json`, though SHOULD send `accept: application/expo+json, application/json` with the order following the preference ordering for the accept header specified in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-5.3.2).
 
+A conformant client library configured to perform [code signing](#code-signing) verification SHOULD also send a `expo-expects-signature` header to indicate that it expects the conformant server to include the `expo-signature` header in the manifest response.
+
 Example:
 ```
 accept: application/expo+json, application/json
 expo-platform: *
 expo-runtime-version: *
+expo-expects-signature: *
```

Comment:
```suggestion
expo-expects-signature-version: *
```
would allow versioning as suggested by @ide without the need for an extra header.